tests/notifications_python_client/test_notifications_api_client.py

```diff
@@ -132,6 +132,23 @@ def test_create_sms_notification_with_personalisation(notifications_client, rmoc
     }
 
 
+def test_create_sms_notification_with_sms_sender_id(notifications_client, rmock):
```

Comment:
Add a test to test for an invalid sender_id?

Reply:
Leaving this as discussed to keep things consistent with the rest of the code and the tests in the other clients